modules/yieldlabBidAdapter.js

```diff
@@ -142,6 +142,13 @@ export const spec = {
           }
         }
 
+        if (isNative(bidRequest, adType)) {
+          const url = `${ENDPOINT}/d/${matchedBid.id}/${bidRequest.params.supplyId}/?ts=${timestamp}${extId}${gdprApplies}${gdprConsent}${pvId}`
+          bidResponse.adUrl = url
+          bidResponse.mediaType = NATIVE
+          bidResponse.native = matchedBid.native
```

Comment:
The native response will be processed by publishers and we can forward the response directly to them. It is already has `json` format

Reply:
Unfortunately the code in line 149 won't work as intended! The respective code needed would look like this:
`const nativeImageAsset = matchedBid.native.assets.find(e => e.id == 2) ? matchedBid.native.assets.find(e => e.id == 2).img : {url: "", w: 0, h: 0};
bidResponse.native = {
    title: matchedBid.native.assets.find(e => e.id == 1) ? matchedBid.native.assets.find(e => e.id == 1).title.text : "",
    body: matchedBid.native.assets.find(e => e.id == 3) ? matchedBid.native.assets.find(e => e.id == 3).data.value : "",
    image: {
        url: nativeImageAsset.url,
        width: nativeImageAsset.w,
        height: nativeImageAsset.h,
    },
    clickUrl: matchedBid.native.link.url,
    impressionTrackers:  matchedBid.native.imptrackers,
};`

Reply:
I added one extra test to cover the default values in native response.